### DISCUSSION TITLE
Infinite crash when specific podcast is first in active queue

### DISCUSSION THREAD

**Discussion by gitfred773202** [XilinJia/Podcini#107] [Q&A] [answered]
I have 1 specific podcast that always crashes Podcini during playback.  URL: https://www.cbc.ca/podcasting/includes/quirksaio.xml
What I can do to reproduce the error:
- Press play on the episode from the subscriptions area
- Press play on the episode from the queue area
- Allow the queue to "roll over" to it next in queue
- If this is the active audio, Podcini crashes
What I've been able to do to get through the infinite crash:
1. Crash Podcini in any way above until the Android audio on the lock screen appears
2. Play audio from the lock screen (the drop down audio player from an unlocked screen does not work)
- Opening Podcini while playing and aborts the playback and crashes
- Opening the App Info(via android) and clicking "Additional settings in the app" does not crash, but there are no audio options/ queue view etc. within this settings area.
After playing the entire podcast via the workaround, the episode will leave the queue, and everything goes back to normal operation within.  This has happened with all episodes I've listened to (5 total(maybe more)) since the move to Podcini.  I've used the last 15(incremental) or more version of the app and have upgraded from Android 14 to 15.

**Comment by XilinJia** [comment]
I'm not too sure how to reproduce it. I got the podcast.

So the steps should be:

> Press play on the episode from the subscriptions area

then you stop it?

> Press play on the episode from the queue area

> Allow the queue to "roll over" to it next in queue

what exactly does this mean?

> If this is the active audio, Podcini crashes
 
what does this mean?

And you found only this podcast crashing Podcini?  Other podcasts seem to work fine for you?

**Reply by gitfred773202** [reply]
If I play any episode from that xml I get a crash.  The only way I can successfully play it is to manipulate the android lock screen player.
If you add it and get the most recent episode, does it crash on play from the queue?

**Reply by gitfred773202** [reply]
I never have to stop it. Actually, I cannot stop it from within Podcini, it crashes.  Simply playing the file causes a crash.
"If this is the active audio, Podcini crashes" means even if this podcast is paused and i open podcini, it crashes.
I'm able to listen to any of the other podcasts I listen to, including others by the CBC.ca.

**Comment by XilinJia** [comment]
Wow, that's very strange. what version of Podcini are you using?
I tried to play some episodes from the podcast (after I subscribed to it and downloaded some episodes) and had no problem.

**Reply by gitfred773202** [reply]
Latest version 6.12.2, I've tried on many versions.  I did a full export, completely cleared data & cache and uninstalled... Reinstalled and imported everything back in.  Same problem with the same podcast.  It is odd indeed.

**Comment by XilinJia** [comment]
And when an episode is the active one, on starting Podcini, it didn't crash.
I tested with the new 6.12.3

**Reply by gitfred773202** [reply]
I've tried with 6.9.3, 6.10.0, 6.11.0, 6.11.3, 6.11.4, 6.11.5, 6.12.0, 6.12.1, 6.12.2,

**Comment by XilinJia** [comment, marked as answer]
Could you do a test when you re-install podcini, don't import anything, but only subscribe to that podcast andsee what happens?

**Reply by gitfred773202** [reply]
This is embarrassing, but I simply removed it and re-added it and it seems to work now.  Are you interested in my OPML file for further troubleshooting?
[podcini-feeds-2024-09-10.zip](https://github.com/user-attachments/files/17513566/podcini-feeds-2024-09-10.zip)

**Comment by XilinJia** [comment]
I see.  I will try you OPML file.  I should have asked you to send a bug report (you know when Podcini crashes, you can get it from Settings -> Bug report).  Please do that when it happens next time. That could be more helpful.
